util/file.go

```diff
@@ -15,6 +15,27 @@ func FileExists(path string) bool {
 	return true
 }
 
+func IsDirectory(path string) bool {
+	stat, err := os.Stat(path)
+	if err != nil {
+		return false
+	}
+	return stat.IsDir()
+}
+
+func ListDirectory(path string) []string {
+	var fileNames []string
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		return nil
+	}
+	for _, file := range files {
+		var fullname = fmt.Sprintf("%s/%s", path, file.Name())
+		fileNames = append(fileNames, fullname)
+	}
+	return fileNames
+}
```

Comment:
This function is not recursive. How about using `filepath.Walk()`?

https://golang.org/pkg/path/filepath/#Walk

Reply:
That's indeed one of the limitations of the current implementation. I'll look into it soon.

Reply:
@cubicdaiya, pushed changes, thanks for the suggestion.